libs/components/code-examples/src/lib/modules/lookup/selection-modal/add-item/example.component.spec.ts

```diff
@@ -6,10 +6,10 @@ import { SkySelectionModalHarness } from '@skyux/lookup/testing';
 import { of } from 'rxjs';
 
 import { LookupSelectionModalAddItemExampleComponent } from './example.component';
-import { DemoService } from './example.service';
+import { ExampleService } from './example.service';
```

Comment:
The `@skyux/code-examples` examples use "example" instead of "demo" for consistency; this rename was missed in the initial pass.